src/frontends/pytorch/src/utils.cpp

```diff
@@ -331,6 +331,16 @@ std::shared_ptr<ov::op::util::FrameworkNode> cast_fw_node(std::shared_ptr<Node>
     return fw_node;
 }
 
+std::shared_ptr<ov::op::util::FrameworkNode> make_list_construct(const ov::OutputVector &inputs) {
+    auto list_construct = std::make_shared<::ov::op::util::FrameworkNode>(inputs, inputs.size());
+    ov::op::util::FrameworkNodeAttrs attrs;
+    attrs.set_type_name("PTFrameworkNode");
+    attrs["PtTypeName"] = "prim::ListConstruct";
```

Comment:
Please use FrameworkNode value instead of "PtTypeName"

Reply:
Done